src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-non-multilingual.sch

```diff
@@ -504,12 +522,10 @@
       <sch:let name="language_present" value="geonet:values-in($language,
               ('eng', 'fra', 'spa', 'zxx'))"/>
 
-      <sch:let name="locMsgCt" value="geonet:prependLocaleMessage($loc/strings/ResourceDescriptionContentType, concat(gmd:CI_OnlineResource/gmd:linkage/gmd:URL, ' : '))" />
+      <sch:let name="resourceContentTypesListMain" value="geonet:resourceContentTypesList($thesaurusDir,$mainLanguage2char)"/>
+      <sch:let name="locMsgCt" value="geonet:prependLocaleMessage(geonet:appendLocaleMessage($loc/strings/ResourceDescriptionContentType, $resourceContentTypesListMain),  concat(gmd:CI_OnlineResource/gmd:linkage/gmd:URL, ' : '))"/>
 
-      <sch:assert test="($contentType = 'Web Service' or $contentType = 'Service Web' or
-              $contentType = 'Dataset' or $contentType = 'Données' or
-              $contentType = 'API' or $contentType = 'Application' or
-              $contentType='Supporting Document' or $contentType = 'Document de soutien')">$locMsgCt</sch:assert>
+      <sch:assert test="$contentType = tokenize($resourceContentTypesListMain, ', ')">$locMsgCt</sch:assert>
```

Comment:
The changes work, but why not do the value check with the thesaurus content instead of the message?

I would update the check to something like this, to unify this kind of thesaurus value checks:

https://github.com/metadata101/iso19139.ca.HNAP/pull/399/files#diff-7c8ff55e3e815d4f6aa802798f973211c9eab945cc11091e8795758a6b60a1deR424-R430

or 

https://github.com/metadata101/iso19139.ca.HNAP/blob/b1fe3249792999af13a0165120906116b2cf736c/src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch#L224-L227